Assets/MixedRealityToolkit/Definitions/InputSystem/MixedRealityGlobalInputEventSystemProfile.cs

```diff
@@ -0,0 +1,204 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Configuration profile settings for setting configuring global input event responses
+    /// </summary>
+    [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Mixed Reality Global Input Event System Profile", fileName = "MixedRealityGlobalInputEventSystemProfile", order = (int)CreateProfileMenuItemIndices.InputActions)]
+    public class MixedRealityGlobalInputEventSystemProfile : BaseMixedRealityProfile, IMixedRealityGlobalInputEventSystem
+    {
+        public virtual void RaiseSourceDetected(IMixedRealityInputSource source, IMixedRealityController controller = null)
```

Comment:
Profiles are only used to store data, any logic should be contained in the service/system.

Reply:
I looked into making it a service - however it doesn't need to have a lifecycle, which is mandated by that system.

MixedRealityGlobalInputEventSystemProfile is meant to be an extensible template. 
In my application, I can expose events to the editor if I choose to do so. 

In practice, it is easier to use and extend as a profile.